### DISCUSSION TITLE
Problems with Problems bar

### DISCUSSION THREAD

**Discussion by elderkeltir** [microsoft/vscode-cpptools#7198] [Q&A] [answered]
Hello!

First of all, let me thank you for such an amazing job with cpp tools!

Now back to business :^)
I have couple small issues with Problems tab:
![image](https://user-images.githubusercontent.com/1653123/111866591-3ba95a00-8977-11eb-889d-cf421dec61d8.png)
**Green** - I don't really need these errors, only two bottom errors, in my Output window there is line during build process: 
`[build] 2 errors generated.`
That are errors I would like to observe in my problems tab. And even more, when I accidentally click on one of green errors I got the annoying pop up at the right side, notifying me that some file is missing. Here is the full text 
`Unable to open 'build.make': Unable to read file '/home/ami/code/HurricaneEngine3D/build/gmake[2]: *** [VkRender/CMakeFiles/VkRender.dir/build.make' (Error: Unable to resolve non-existing file '/home/ami/code/HurricaneEngine3D/build/gmake[2]: *** [VkRender/CMakeFiles/VkRender.dir/build.make').`
Is there any option to disable this and leave only useful errors listed?

**Purple** - Why it states GCC if I configured it to compile code with clang? 
```
{
    "configurations": [
        {
            "name": "Linux",
            "includePath": [
                "${workspaceFolder}/**"
            ],
            "defines": [],
            "compilerPath": "/usr/bin/clang",
            "cStandard": "c99",
            "cppStandard": "c++17",
            "intelliSenseMode": "linux-clang-x64",
            "configurationProvider": "ms-vscode.cmake-tools",
            "compileCommands": "${workspaceFolder}/build/compile_commands.json"
        }
    ],
    "version": 4
}
```

Any help is appreciated!

**Comment by sean-mcmanus** [comment, marked as answer]
None of those errors appear to be from our C/C++ extension. Are you able to identify which extension those are coming from? Our extension only shows "C/C++" and "gcc" problem types (the "gcc" is just the name of the problem matcher and applies to clang as well), but in your case is "GCC" (capitalized) so I'm not sure where that is getting defined, but the same reasoning probably still applies.

![image](https://user-images.githubusercontent.com/19859882/111977544-15430600-8ac0-11eb-83c4-56e93e55a62a.png)

**Reply by elderkeltir** [reply]
Thanks for the reply!
It may be an error generated by cmake-tools, will check that.
